pkg/webservice/dao/partition_info.go

```diff
@@ -24,12 +24,29 @@ type PartitionDAOInfo struct {
 	Queues        QueueDAOInfo      `json:"queues"`
 }
 
+type PartitionInfo struct {
+	Name                    string            `json:"name"`
+	Capacity                PartitionCapacity `json:"capacity"`
```

Comment:
This capacity will be the configured capacity one, correct?

Reply:
As and when any new node gets added, Node's total resource would get added to this capacity. Similarly, for node removal resource would be subtracted from this capacity.

Reply:
ok. thanks